staging/src/k8s.io/kubectl/pkg/cmd/exec/exec.go

```diff
@@ -107,19 +110,26 @@ func NewCmdExec(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	cmd.Flags().BoolVarP(&options.Stdin, "stdin", "i", options.Stdin, "Pass stdin to the container")
 	cmd.Flags().BoolVarP(&options.TTY, "tty", "t", options.TTY, "Stdin is a TTY")
 	cmd.Flags().BoolVarP(&options.Quiet, "quiet", "q", options.Quiet, "Only print output from the remote session")
+	cmd.Flags().DurationVar(options.PingPeriod, "ping", *options.PingPeriod, "The period between pings, to prevent the connection from becoming idle")
```

Comment:
```suggestion
	cmd.Flags().DurationVar(options.PingPeriod, "ping", *options.PingPeriod, "The period between pings, to prevent the connection from becoming idle (default is 5seconds, use `0` to disable keep alive pings)")
```